sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -74,7 +90,7 @@ Authenticating user accounts is the easiest way to get started with minimal set
 
 ### Authenticating Service Principals
 
-An Azure service principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources. This access is restricted by the roles assigned to the service principal, giving you control over which resources can be accessed and at which level. For security reasons, it's always recommended to use service principals with automated tools rather than allowing them to log in with a user identity.
+An Azure service principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources. The roles assigned by the service principal will determine what resources are accessible. For security reasons, we recommended using service principals through automation rather than allowing them to log in with a user identity.
```

Comment:
```suggestion
An Azure service principal is an identity created for use with applications, hosted services, and automated tools to access Azure resources. The roles assigned by the service principal will determine what resources are accessible. For security reasons, use service principals through automation rather than allowing them to log in with a user identity.
```

Reply:
Thank you!